Coalition: Pilgrimage (#6076)
endless-sky/endless-sky: data/coalition/coalition missions.txt

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

mission "Coalition: Pilgrimage 1"
	minor
	name "Shrine of the World Shaper"
	description "Bring Vellorae to <destination>, where she will begin her mourning pilgrimage for her late husband."
	passengers 1
	to offer
		has "Coalition: Contributor: done"
	source "Bright Echo"
	destination "Saros"
	on offer
		conversation

> 			`Shortly after entering the spaceport, you catch the attention of a Saryd lady with gray, braided hair that reaches to her shoulders. She drags along a single suitcase. "Hello Captain. Taking new passengers, are you?" she asks. She shows you a piece of jewelry on her hand, a small bronze bangle with a diamond at the center. "Vellorae, my name is, and in need of a transport for a pilgrimage, I am."`

Author: Google gives me `jewelry` as the American spelling, and ES uses the American spelling.

> 			`Shortly after entering the spaceport, you catch the attention of a Saryd lady with gray, braided hair that reaches to her shoulders. She drags along a single suitcase. "Hello Captain. Taking new passengers, are you?" she asks. She shows you a piece of jewelry on her hand, a small bronze bangle with a diamond at the center. "Vellorae, my name is, and in need of a transport for a pilgrimage, I am."`

Reviewer: Oops, sorry about that.

			`	Once the sun peeks over the tree tops and into the clearing, the visitors move away from the pillars and the alter while the couples line up and walk to the altar one at a time. There they remain for a few minutes, speaking to one another, before moving aside for the next pair. "Exchanging their vows, they are," Vellorae explains. "Our most popular wedding locations, the shrines here on the capital are."`
			choice
				`	"I thought this was a shrine for mourning the dead?"`
					goto shrine
				`	"Popular? In the middle of the woods, by some rocks?"`
			`	She looks at you in confusion, then chuckles. "The creator of all forests, Aul Kell is, so naturally inside forests, his shrines are. Odd, I suppose they may look, to one outside our culture."`
				goto ritual
			label shrine
			`	"Attributed with the gift of sunlight, Aul Kell is. As such, seek his blessings in the shrines, for many occasions we do. Only some of them, mourning and espousals are."`
			label ritual

> 			`	The couples keep exchanging their vows, until just before midday, when the sun is nearly over the central altar. At this point, the Saryds all look around the clearing, making space for Vellorae as they see her approaching the altar. She places the bangle atop the altar so it catches the sun's light, reflecting it throughout the clearing. Vellorae then begins walking around the altar, repeatedly circling it as the sun climbs higher, until it finally reaches its peak and the sun's reflection on the gem is strongest. At this point she approaches the altar again and covers the bangle with her hands. She remains there silently for a moment, then she retrieves the bangle and leaves the shrine. As she walks past, the other Saryds give her sympathetic looks, and one of the younger ones even approaches to offer her condolences before Vellorae approaches you again.`

Reviewer: I wanted to reduce the usage of 'altar' so that it doesn't get overused.
```suggestion
			`	The couples keep exchanging their vows, until just before midday, when the sun is nearly over the central altar. At this point, the Saryds all look around the clearing, making space for Vellorae as they see her approaching the shrine. She places the bangle atop the altar so it catches the sun's light, reflecting it throughout the clearing. Vellorae then begins walking around the altar, repeatedly circling it as the sun climbs higher, until it finally reaches its peak and the sun's reflection on the gem is strongest. At this point she approaches the altar again and covers the bangle with her hands. She remains there silently for a moment, then she retrieves the bangle and leaves the shrine. As she walks past, the other Saryds give her sympathetic looks, and one of the younger ones even approaches to offer her condolences before Vellorae approaches you again.`
```

				goto end
			label skip
			`	You wait for a few hours by the spaceport city, until Vellorae returns about an hour after noon.`
			label end
			`	"<destination>, the next stop is. Wish to give Adfices Velaulra's Blessing, I wish." She says, heading into her bunk, as you prepare for the next trip.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Vellorae hasn't entered the system yet. Better depart and wait for it.`
	on complete
		"coalition jobs" ++
		payment 49300

mission "Coalition: Pilgrimage 3"
	landing
	name "Visit the Witch of Dusk"
	description "Bring Vellorae to <destination> so that she may continue her mourning pilgrimage for her late husband."
	passengers 1
	to offer
		has "Coalition: Pilgrimage 2: done"
	source "Flowing Fields"
	destination "Secret Sky"
	on offer
		conversation
			`Vellorae directs you to a part of the planet cut by a long, thin river, pays you 49,300 credits once you've landed at one of the available hangars in a larger village, and sets out for a marble rotunda by the river banks.`

> 			`	Following her there, you see Saryd mothers playing with their children and others holding young infants or carefully bathing them in the waters. As the two of you step into the rotunda, you notice the painting on the ceiling, a Saryd woman laying in a field, watching over several children playing around her.`

Reviewer: ```suggestion
			`	Following her there, you see some Saryd mothers playing with their children. Others hold young infants or carefully bath them in the waters. As the two of you step into the rotunda, you notice the painting on the ceiling, a Saryd woman laying in a field, watching over several children playing around her.`
```

			choice
				`	"If this place is a shrine, what's with all the kids playing around as if it was a park?"`
				`	"Sorry, I'm not one for prayers."`
					goto noprayer
				`	"Of course. What do I have to do?"`
					goto prayer
			`	"A deity of life and plenty, Velaulra is, and as the greatest bounty of all, life is, strong ties with newborn children, she has. Common places for mothers to bond with their young ones, her shrines are." Vellorae explains. "In my case, here to thank her for Adfices' life, and all plenty there was in it, I am. Take my hands for a prayer, would you?"`
			choice
				`	"I know this is important for you, but I'm not one for prayers, sorry."`
				`	"I don't fully understand this, but I'd be glad to. What do I have to do?"`
					goto prayer
			label noprayer
			`	"I see. That's fine, understand it, I do." She holds the bangle in one hand, covering it with the other, and closes her eyes for just under a minute while standing there in silence.`
				goto river
			label prayer

> 			`	"Just over mine, place yours." She gestures with her hands, and you both place them over the bangle. She closes her eyes, and you do the same. You both stand still like that for just under a minute, until you feel her move away.`

Reviewer: ```suggestion
			`	"Just over mine, place yours." She gestures with her hands, and you both place them over the bangle. She closes her eyes as you do the same. You both stand still like that for just under a minute until you feel her move away.`
```

> 			`	"Just over mine, place yours," she gestures with her hands, and you place your own over the bangle. She closes her eyes, and you do the same, standing still like that for just under a minute, until you feel her moving away.`

Reviewer: ```suggestion
			`	"Just over mine, place yours," she gestures with her hands, and you place your own over the bangle. She closes her eyes, and you do the same. You both stand still like that for just under a minute, until you feel her move away.`
```

			label river

> 			`	After the prayer, Vellorae leaves the rotunda and heads to the river bank, where she lays close to the waters and lowers the bangle to start washing it in the current in a slow and clearly ceremonial manner. The Saryd children watch her curiously, some edging closer before being stopped by their mothers. Once Vellorae is done, she places the bangle, still wet from the river,  on her hand again and tells you that she's done at this shrine.`

Reviewer: ```suggestion
			`	After the prayer, Vellorae leaves the rotunda and heads to the river bank, where she lays close to the waters and lowers the bangle to start washing it in the current in a slow and clearly ceremonial manner. The Saryd children watch her curiously, some edging closer before being stopped by their mothers. Once Vellorae is done, she places the bangle, still wet from the river, on her hand again and tells you that she's done at this shrine.`
```

			`	You head back to your ship, where she tells you she'd like to stop at <destination> next. "Visit Magl Ina next, I must." She heads to her bunk, and you go to your cockpit to set course for <planet>.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Vellorae hasn't entered the system yet. Better depart and wait for it.`
	on complete
		"coalition jobs" ++
		payment 45500

mission "Coalition: Pilgrimage 4"
	landing
	name "Master Hunter's Path"
	description "Bring Vellorae to <destination> so that she may continue her mourning pilgrimage for her late husband."
	passengers 1
	to offer
		has "Coalition: Pilgrimage 3: done"
	source "Secret Sky"
	destination "Shadow of Leaves"
	on offer
		conversation
			`<origin>'s spaceport searchlights guide you to the hangars in the middle of a thunderstorm, and Vellorae says she'll wait it out inside your ship as she pays you 45,500 credits. "At her shrine only after dusk, Magl Ina is anyway, so in a hurry, I am not." The storm lasts halfway into the evening, and once the light reaching through the layers of fog starts to dim, you open your ship's hatch for Vellorae to head out.`

> 			`	You accompany her across the murky spaceport, until you find a local Saryd for her to talk to, and Vellorae asks him if he would guide the two of you to the shrine. "To a site of worship, why is this human going?" He asks, while eyeing you somewhat suspiciously.`

Reviewer: ```suggestion
			`	You accompany her across the murky spaceport, until you find a local Saryd for her to talk to. Vellorae asks him if he would guide the two of you to the shrine. "To a site of worship, why is this human going?" he asks, while eyeing you somewhat suspiciously.`
```

			`	"On a pilgrimage to mourn my late husband, I am." Vellorae says. "Helping me with traveling to each shrine, Captain <last> is."`
			`	"But, to the shrine, must the Captain go?" He asks. "Help you with mourning there, a human cannot."`
			`	"Helping me in this journey, the kind Captain has been, so mind it, I do not. Like to come with me to the shrine, would you, Captain?"`
			choice
				`	"If it's alright for me to go, I'd love to."`
				`	"I don't want to disturb anyone, so I'll stay by the ship and wait for you."`
					goto notgo

> 			`	"Wonderful!" She cheers, tapping her frontal hooves lightly on the ground. "Now, guide us to the shrine, would you, young dear?" She asks the local Saryd, who turns around and gestures for you to follow.`

Reviewer: ```suggestion
			`	"Wonderful!" She cheers, tapping her frontal hooves lightly on the ground. "Now, guide us to the shrine, would you, young dear?" she asks the local Saryd, who turns around and gestures for you to follow.`
```

> 			`	You stick close to Vellorae as the two of you try to keep the local Saryd in sight. The task is difficult as he nearly vanishes into the fog whenever he gets too far ahead, only for his silhouette to become clearer again after he waits for you to catch up. After many minutes you feel the ground beneath your shoes shift from the solid spaceport platform, to gravel, and then to wet patches of the planet's mossy "grass." Finally you reach a minuscule lake, or perhaps a very large puddle, in the middle of the mossy field surrounded by toppled and hollowed out tree trunks.`

Reviewer: ```suggestion
			`	You stick close to Vellorae as the two of you try to keep the local Saryd in sight. The task is difficult as he nearly vanishes into the fog whenever he gets too far ahead, only for his silhouette to become clearer again after he waits for you to catch up. After many minutes you feel the ground beneath your shoes shift from the solid spaceport platform, to gravel, and then to wet patches of the planet's mossy "grass." Finally you reach a minuscule lake, or perhaps a very large puddle, in the middle of a mossy field surrounded by toppled and hollowed out tree trunks.`
```

> 			`	You stick close to Vellorae as the two of you try to keep the local Saryd in sight. The task is difficult as he nearly vanishes into the fog whenever getting too far ahead, only for his silhouette to become clearer again after he waits for you to catch up. After many minutes you feel the ground beneath your shoes shift from the solid spaceport platform, to gravel, and then to wet patches of the planet's mossy "grass." Finally you reach a minuscule lake, or perhaps a very large puddle, in the middle of the mossy field surrounded by toppled and hollowed out tree trunks.`

Reviewer: ```suggestion
			`	You stick close to Vellorae as the two of you try to keep the local Saryd in sight. The task is difficult as he nearly vanishes into the fog whenever he gets too far ahead, only for his silhouette to become clearer again after he waits for you to catch up. After many minutes you feel the ground beneath your shoes shift from the solid spaceport platform, to gravel, and then to wet patches of the planet's mossy "grass." Finally you reach a minuscule lake, or perhaps a very large puddle, in the middle of the mossy field surrounded by toppled and hollowed out tree trunks.`
```

> 			`	You stick close to Vellorae as the two of you try to keep the local Saryd in sight, though it is difficult as he nearly vanishes into the fog whenever getting too far ahead, only for his silhouette to become clearer again after he waits for you to catch up to. After many minutes you feel the ground beneath your shoes shift from the solid spaceport platform to gravelly ground and then to wet patches of the planet's mossy "grass." Finally you reach a minuscule lake, or perhaps a very large puddle, in the middle of the mossy field which is surrounded by toppled and hollowed out tree trunks.`

Reviewer: ```suggestion
			`	You stick close to Vellorae as the two of you try to keep the local Saryd in sight. The task is difficult as he nearly vanishes into the fog whenever getting too far ahead, only for his silhouette to become clearer again after he waits for you to catch up. After many minutes you feel the ground beneath your shoes shift from the solid spaceport platform, to gravel, and then to wet patches of the planet's mossy "grass." Finally you reach a minuscule lake, or perhaps a very large puddle, in the middle of the mossy field surrounded by toppled and hollowed out tree trunks.`
```

> 			`	As the two of you approach the water, the local Saryd warns you not to touch it. Vellorae looks at it as if gazing into a mirror or, perhaps, just trying to make out the reflection of the moon, scattered and vague though it is through the layers of mist. She then moves to one of the tree trunks, and placing the bangle inside it as she lays on the ground facing the water, places both her hands reverently on the trunk. She spends nearly an hour in this position, during which the chill of the fog slowly seeps through your coat and into your bones. Vellorae and the Saryd who guided you don't seem particularly bothered by it, but she is quick to finish once she looks over and notices you shivering. She picks the bangle up from the trunk and stands, then asks the other Saryd to guide you two back to the spaceport.`

Reviewer: ```suggestion
			`	As the two of you approach the water, the local Saryd warns you not to touch it. Vellorae looks at it as if gazing into a mirror; or perhaps she is just trying to make out the reflection of the moon, scattered and vague though it is through the layers of mist. She then moves to one of the tree trunks, and placing the bangle inside it as she lays on the ground facing the water, lays both her hands reverently on the trunk. She spends nearly an hour in this position, during which the chill of the fog slowly seeps through your coat and into your bones. Vellorae and the Saryd who guided you don't seem particularly bothered by it, but she is quick to finish once she looks over and notices you shivering. She picks the bangle up from the trunk and stands, then asks the other Saryd to guide you two back to the spaceport.`
```

> 			`	As the two of you approach the water, the local Saryd warns you not to touch it, and Vellorae looks at it as if gazing into a mirror or, perhaps, just trying to make out the reflection of the moon, scattered and vague though it is through the layers of mist. She then moves to one of the tree trunks, and placing the bangle inside it as she lays on the ground facing the water, places both her hands reverently on the trunk. She spends nearly an hour in this position, during which the chill of the fog slowly seeps through your coat and into your bones. Vellorae and the Saryd who guided you don't seem particularly bothered by it, but she is quick to finish once she looks over and notices you shivering. She picks the bangle up from the trunk and stands, then asks the other Saryd to guide you two back to the spaceport.`

Reviewer: ```suggestion
			`	As the two of you approach the water, the local Saryd warns you not to touch it. Vellorae looks at it as if gazing into a mirror or, perhaps, just trying to make out the reflection of the moon, scattered and vague though it is through the layers of mist. She then moves to one of the tree trunks, and placing the bangle inside it as she lays on the ground facing the water, places both her hands reverently on the trunk. She spends nearly an hour in this position, during which the chill of the fog slowly seeps through your coat and into your bones. Vellorae and the Saryd who guided you don't seem particularly bothered by it, but she is quick to finish once she looks over and notices you shivering. She picks the bangle up from the trunk and stands, then asks the other Saryd to guide you two back to the spaceport.`
```

				goto end
			label notgo
			`	She sighs briefly, and agrees to meet with you later, following the Saryd local deep into the fog stretching out of the city. You head inside your ship, waiting for a few hours until you hear her voice just outside your hatch. You open the hatch, and let her in.`
			label end
			`	"To <destination>, we must go now. The Path of Lor Ie Vees, I must take," she says, heading to her bunk to rest.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Vellorae hasn't entered the system yet. Better depart and wait for it.`
	on complete
		"coalition jobs" ++
		payment 69700

mission "Coalition: Pilgrimage 5"
	landing
	name "Woodland Thing"
	description "Bring Vellorae to <destination>, where she will finish her mourning pilgrimage."
	passengers 1
	to offer
		has "Coalition: Pilgrimage 4: done"
	source "Shadow of Leaves"
	destination "Cold Horizon"
	on offer
		conversation

> 			`Vellorae hands you 69,700 credits as you land, and says she'll head to the "path" once the sun sets. "Come to this world for our honeymoon, Adfices and I did, and visited the path, we had. Know it's starting point, I already do."`

Reviewer: ```suggestion
			`Vellorae hands you 69,700 credits as you land, and says she'll head to the "path" once the sun sets. "Come to this world for our honeymoon, Adfices and I did, and visited the path, we had. Know it's starting point, I already do."`
```
```suggestion
			`Vellorae hands you 69,700 credits as you land, and says she'll head to the "path" once the sun sets. "Come to this world for our honeymoon, Adfices and I did, and visited the path, we had. Know its starting point, I already do."`
```

> 			`	She tells you she'll pass time until then by visiting local shops, asking if you'd like to come with her.`

Reviewer: ```suggestion
			`	She tells you she'll pass the time until then by visiting local shops, asking if you'd like to come with her.`
```

			choice
				`	(Go along with her.)`
				`	(Stay in your ship.)`
					goto notgo
			`	She takes you around the spaceport city, touring some clothing shops and bringing you to some parks she tells you she had visited with her husband. Shortly after noon you two enter a restaurant, where she insists on paying for your meal. You tour the city with her for a while longer and, just before the sun sets, reach the forest at the base of the mountain on top of which Starlit University is located.`

> 			`	Once night falls, many lights start to appear inside the forest, and Vellorae beckons you to follow her into the forest. "Stick close, you should. Difficult to navigate the path, for other species it often is." As you walk down a trail with her, you approach one of the lights to see that it's a bonfire on a platform in a miniature clearing, at what you hope is a safe distance from the trees around it. Continuing to the next light, Vellorae clutches the bangle in her hands the entire time, humming some guttural, almost tribal chant. As you pass by another fire, a short burst of flames gets closer to some nearby shrubs, but thankfully is put out before it can cause any harm, extinguished with an odd, large fabric net on a pole by a cloaked Saryd who seemed to materialize from the trees near the fire.`

Reviewer: ```suggestion
			`	Once night falls, many lights start to appear inside the forest, and Vellorae beckons you to follow her into the forest. "Stick close, you should. Difficult to navigate the path, for other species it often is." As you walk down a trail with her, you approach one of the lights to see that it's a bonfire on a platform in a miniature clearing, at what you hope is a safe distance from the trees around it. Continuing to the next light, Vellorae clutches the bangle in her hands the entire time, humming some guttural, almost tribal chant. As you pass by another fire, a short burst of flames gets closer to some nearby shrubs, but thankfully is put out before it can cause any harm. It is extinguished with an odd, large fabric net on a pole by a cloaked Saryd who seemed to materialize from the trees near the fire.`
```

> 			`	Once night falls, many lights start to appear inside the forest, and Vellorae beckons you to follow her into the forest. "Stick close, you should. Difficult to navigate the path, for other species it often is." A you walk down a trail with her, you approach one of the lights to see that it's a bonfire on a platform in a miniature clearing, at what you hope is a safe distance from the trees around it. Continuing to the next light, Vellorae clings the bangle in her hands the entire time, humming some guttural, almost tribal chant. As you pass by another fire, a short burst of flames gets closer to some nearby shrubs, but thankfully is put out before it can cause any harm, extinguished with an odd, large fabric net on a pole by a cloaked Saryd who seemed to materialize from the trees near the fire.`

Reviewer: ```suggestion
			`	Once night falls, many lights start to appear inside the forest, and Vellorae beckons you to follow her into the forest. "Stick close, you should. Difficult to navigate the path, for other species it often is." As you walk down a trail with her, you approach one of the lights to see that it's a bonfire on a platform in a miniature clearing, at what you hope is a safe distance from the trees around it. Continuing to the next light, Vellorae clutches the bangle in her hands the entire time, humming some guttural, almost tribal chant. As you pass by another fire, a short burst of flames gets closer to some nearby shrubs, but thankfully is put out before it can cause any harm, extinguished with an odd, large fabric net on a pole by a cloaked Saryd who seemed to materialize from the trees near the fire.`
```

			choice
				`	"Why? Is that some god of knowledge?"`
					goto knowledge
				`	"Maybe they like being on good terms with the divine in hopes of doing well in tests?"`
			`	She stops for a second to laugh. "The case, that might be." She looks around, as if seeing if someone else caught her breaking the serene, humming chant.`
			label knowledge
			`	"Taught the first Saryds how to hunt, he did. To pursue knowledge, and overcome challenges, his teachings are." She explains. "Seek communion with him, not only students, but all going through a difficult period, should."`
			`	You continue to follow her closely through the forest, regularly passing more bonfires, until you see multiple lights in the distance and realize it is the edge of the forest leading back into the city. Only as you step out of the woods do you notice that you're back where the path started, even though you were certain you had kept to only one side of the mountain.`
			`	Vellorae takes a final look at the path, where only a few of the bonfires' lights are still visible through the dense groupings of pine trees. She tells you she's ready to leave, and you bring her back to your ship.`
				goto end
			label notgo

> 			`	She says she'll be back at night and leaves. You watch a few transmissions from the planet, most of them in some way related to Starlit University, though one is what you guess is a Saryd comedy movie, something about one who cut down trees in order to make scarves but came to regret it later. You try and follow along, but subtitles for the transmissions only come in Kimek or Arach. A few hours into the night, you leave your ship to wait for Vellorae, who arrives shortly after.`

Reviewer: ```suggestion
			`	She says she'll be back at night and leaves. You watch a few transmissions from the planet, most of them in some way related to Starlit University. One is what you guess is a Saryd comedy movie, something about one who cut down trees in order to make scarves but came to regret it later. You try and follow along, but subtitles for the transmissions only come in Kimek or Arach. A few hours into the night, you leave your ship to wait for Vellorae, who arrives shortly after.`
```

				`	"Is something wrong with the next shrine?"`
			`	She ignores your question, fixated on the bangle for a few moments. "<planet>. To <destination>, we must go." She finally says, before going back to her room.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Vellorae hasn't entered the system yet. Better depart and wait for it.`
	on complete
		"coalition jobs" ++
		payment 80000
		conversation
			`Vellorae hands you <payment> once you land, and asks, "Come with me for this last bit, could you, Captain <last>?"`
			choice
				`	"Sure. Where are we going?"`
					goto shrine
				`	"I'll pass, I'm not up for it right now."`
			`	Her eyes go to the bangle, travel around the room, and then fixate on your again. "Please, Captain? I know that agreed to follow me to the shrines for the job, you didn't, but alone there, I do not wish to be."`
			choice
				`	"Alright, if it'll help you relax I'll go with you."`
					goto shrine
				`	"Sorry, but I'd rather rest in the ship."`
			`	She takes a step back, looking down to the bangle again. "Alright. Yes, your responsibility, this is not. Meet you in the spaceport later, I shall."`

> 			`	You open the hatch for her, and she heads to one of the Saryds nearby and begins to talk to them. A few seconds into the conversation, they tense up, crossing their arms and swiping them down as if refusing something, then running off. She tries again with some of the other Saryds, but continues to be rejected. She looks back at your ship dejectedly, then starts eyeing the other ships docked nearby before heading towards a Kimek Thorn that has just landed. As she talks to the captain, the Kimek seems confused, but after a moment, briefly reenters their ship, returning in a puffy coat to follow Vellorae to the railway hub.`

Reviewer: ```suggestion
			`	You open the hatch for her, and she heads to one of the Saryds nearby and begins to talk to them. A few seconds into the conversation, they tense up, crossing their arms and swiping them down as if refusing something, then running off. She tries again with some of the other Saryds, but continues to be rejected. She looks back at your ship dejectedly, then starts eyeing the other ships docked nearby before heading towards a Kimek Thorn that has just landed. As she talks to the captain, the Kimek seems confused, but after a moment he briefly reenters their ship, returning in a puffy coat to follow Vellorae to the railway hub.`
```

> 			`	You open the hatch for her, and she heads to one of the Saryds nearby, to talk to them. A few seconds into the conversation and they tense up, crossing their arms and swiping them down, as if refusing something, then run off. She tries again with some of the other Saryds, but continues to be rejected. She looks back at your ship dejectedly, then starts eyeing the other ships docked nearby before heading towards a Kimek Thorn that has just landed. As she talks to the captain, the Kimek seems confused, but after a moment, briefly reenters their ship, returning in a puffy coat to follow Vellorae to the railway hub.`

Reviewer: ```suggestion
			`	You open the hatch for her, and she heads to one of the Saryds nearby and begins to talk to them. A few seconds into the conversation, they tense up, crossing their arms and swiping them down as if refusing something, then running off. She tries again with some of the other Saryds, but continues to be rejected. She looks back at your ship dejectedly, then starts eyeing the other ships docked nearby before heading towards a Kimek Thorn that has just landed. As she talks to the captain, the Kimek seems confused, but after a moment, briefly reenters their ship, returning in a puffy coat to follow Vellorae to the railway hub.`
```

			`	Long after the sun has set, the Kimek returns, rushing to their ship while trying to shake off the cold. You ask them about Vellorae, and they tell you she headed for the spaceport to "clear her head."`
				accept
			label shrine

> 			`	She takes a long, deep breath. "Thank you, Captain. Far outside the village, the site is. Come, show you the way there, I will." You leave your ship and follow her to the railway hub and board one of trains that appears to be one of the oldest and more simple at the hub. It lacks seats for non-Saryds, so you are forced to hold tight to what handrails you can reach.`

Reviewer: ```suggestion
			`	She takes a long, deep breath. "Thank you, Captain. Far outside the village, the site is. Come, show you the way there, I will." You leave your ship and follow her to the railway hub and board one of trains that appears to be one of the oldest and most simple. It lacks seats for non-Saryds, so you are forced to hold tight to what handrails you can reach.`
```

> 			`	You remain on the train until the very last stop, when the sun is already starting to set. as you depart the train, you see a rural village with just a few dozen buildings and a simple dirt road, where a couple workers battle the snow and ice to keep it clear and easy to traverse. You follow Vellorae down the road, and once you reach a rocky pillar with a hole carved through it, Vellorae turns to the forest heading out on a thin, straight trail. Looking behind you, you see the setting sun fitting inside the pillar's opening.`

Reviewer: ```suggestion
			`	You remain on the train until the very last stop, when the sun is already starting to set. As you depart the train, you see a rural village with just a few dozen buildings and a simple dirt road, where a couple workers battle the snow and ice to keep it clear and easy to traverse. You follow Vellorae down the road, and once you reach a rocky pillar with a hole carved through it, Vellorae turns to the forest, heading out on a thin, straight trail. Looking behind you, you see the setting sun fitting inside the pillar's opening.`
```

> 			`	As you follow Vellorae down the trail, you see the trees gradually show more and more signs of a forest fire, with darker, damaged barks, increasingly ashy soil, and many fallen, decomposing portions of the forest. The trail ends abruptly at a large scar scorched through the forest. The piles of ash still present below your feet almost resemble the rest of the planet's otherwise snowy ground, but somehow are kept from mixing with the snow. Whether deliberately cleared by more workers or some other force, you cannot tell. Near the center of the scar you see a small mound with an opening in the side, almost like a "shack" made out of the charred tree trunks and rocks. A faint light comes from within it.`

Author: I'll keep it.

> 			`	As you follow Vellorae down the trail, you see the trees gradually show more and more signs of a forest fire, with darker, damaged barks, increasingly ashy soil, and many fallen, decomposing portions of the forest. The trail ends abruptly at a large scar scorched through the forest. The piles of ash still present below your feet almost resemble the rest of the planet's otherwise snowy ground, but somehow are kept from mixing with the snow. Whether deliberately cleared by more workers or some other force, you cannot tell. Near the center of the scar you see a small mound with an opening in the side, almost like a "shack" made out of the charred tree trunks and rocks, and a faint light coming from within it.`

Reviewer: ```suggestion
			`	As you follow Vellorae down the trail, you see the trees gradually show more and more signs of a forest fire, with darker, damaged barks, increasingly ashy soil, and many fallen, decomposing portions of the forest. The trail ends abruptly at a large scar scorched through the forest. The piles of ash still present below your feet almost resemble the rest of the planet's otherwise snowy ground, but somehow are kept from mixing with the snow. Whether deliberately cleared by more workers or some other force, you cannot tell. Near the center of the scar you see a small mound with an opening in the side, almost like a "shack" made out of the charred tree trunks and rocks. A faint light comes from within it.`
```

> 			`	While you're looking at the mound, Vellorae grabs your hand and starts hesitantly pulling you towards it. Inside, the fading sunlight behind you contrasts with the source of the light within, a small pyre in the middle of the "shack" which has only a few lone flames dancing on the embers. Vellorae lays down in front of it, and removes her bangle and places it in the flames for a brief moment before snatching it up and standing again. Once on her feet, she grabs your hand again, tugging you along as she hurries back to the edge of the scorched clearing.`

Reviewer: ```suggestion
			`	While you're looking at the mound, Vellorae grabs your hand and starts hesitantly pulling you towards it. Inside, the fading sunlight behind you contrasts with the source of the light within, a small pyre in the middle of the "shack" which has only a few lone flames dancing on the embers. Vellorae lays down in front of it, removes her bangle, and places it in the flames for a brief moment before snatching it up and standing again. Once on her feet, she grabs your hand again, tugging you along as she hurries back to the edge of the scorched clearing.`
```

			`	She seems to settle again once you have reached the trail, letting go of your hand and slowing her pace.`
			choice
				`	"What was that about? What got you so agitated?"`
				`	(Stay quiet.)`
					goto return

> 			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper road again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, it was, and of the light only specks remained, but finished, it was not. To implore Adfices is kept safe from it, this was."`

Reviewer 2: ```suggestion
			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper road again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, It was, and of the light from It's eyes, only specks remained. But finished, Ensu Uri was not. Still a threat It remains, and to implore Adfices is kept safe from It, this was."`
```

Something like this, then, to help make it clearer?

Author: I'm fine with capitalizing "it", but don't want to name drop again, and the "Still a threat" part isn't necessary.

> 			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper road again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, it was, and of the light only specks remained, but finished, it was not. To implore Adfices is kept safe from it, this was."`

Reviewer: Would this be ok?
```suggestion
			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper road again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, it was, and of the light only specks remained, but completely destroyed, it was not. To implore Adfices is kept safe from it, this ritual was for."`
```

Author: Personally I prefer EjoThims' suggestion, save for what I commented about it. Just waiting for it to be updated.

Reviewer: Alright then.

> 			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper road again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, It was, and of the light from It's eyes, only specks remained. But finished, It was not, so to implore Adfices is kept safe from It, this was."`

Reviewer 3: ```suggestion
			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the rocky pillar and are back on the proper road again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, It was, and of the light from Its eyes, only specks remained. But finished, It was not, so to implore Adfices is kept safe from It, this was."`
```

> 			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper road again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, it was, and of the light only specks remained, but finished, it was not. To implore Adfices is kept safe from it, this was."`

Reviewer: Some of this is unclear to me. What is the 'it' in `Broken apart, it was,`? The eyes of Ensu Uri? And what do you mean by `finished, it was not.`? Was their destruction not completely destroyed? Sorry if I'm missing something obvious here.

Reviewer 2: I wasn't clear on this either, but just assumed it came from my troubles parsing the speech pattern.

Author: "To gift us sunlight, Aul Kell stole the eyes Ensu Uri. It (Ensu Uri) was broken apart, and only specks remained (of what once were its eyes), but it wasn't finished (as in, breaking it apart didn't kill it.)"

> 			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper raid again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, it was, and of the light only specks remained, but finished, it was not. To implore Adfices is kept safe from it, this was."`

Reviewer 4: ```suggestion
			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper way again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, it was, and of the light only specks remained, but finished, it was not. To implore Adfices is kept safe from it, this was."`
```

The word "raid" here doesn't seem to fit. Is it meant to be path? or way?

Reviewer 2: Oops, road.

> 			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper raid again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, it was, and of the light only specks remained, but finished, it was not. To implore Adfices is kept safe from it, this was."`

Reviewer 2: ```suggestion
			`	She doesn't answer your question, instead simply continuing down the trail. Once you pass the the rocky pillar and are back on the proper road again, she finally speaks up. "That... to gift us sunlight, stole the eyes of Ensu Uri, Aul Kell did. Broken apart, it was, and of the light only specks remained, but finished, it was not. To implore Adfices is kept safe from it, this was."`
```

			choice
				`	"Thought you'd be feeling better than this, you've finally finished your pilgrimage after all."`
					goto better
				`	"I get it, sometimes I also need a drink to help me think things over."`
			`	She chuckles, "A herbal tea, this is. Help me calm down and rest on the way back home, it will."`
				goto end
			label better
			`	"Yes, relieved I feel, that able to do this, I was. Just, still coming to terms with everything, I am."`
			label end
			`	She finishes the drink, and gets up to pay for her order, following you to your ship afterwards. "Ready to head back home, I am," she tells you, heading into her bunk, as you prepare the course for <destination>.`
				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying Vellorae hasn't entered the system yet. Better depart and wait for it.`
	on complete
		"coalition jobs" += 3
		payment 220000
		conversation
			`Once she leaves your ship, Vellorae gives you your payment of <payment>, and takes your hands in hers. "Deeply grateful I am, for having accompanied me in this journey, Captain <last>. Sure that grateful for your help, Adfices also is."`

> 			`	She lets go of your hands, and wishes you the best in your travels, as she leaves for her home.`

Reviewer 2: ```suggestion
			`	She lets go of your hands, and wishes you the best in your travels, then turns and leaves for her home.`
```
